webbeans-junit5/src/test/java/org/apache/openwebbeans/junit5/parameter/ParameterResolutionTest.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.openwebbeans.junit5.parameter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.apache.openwebbeans.junit5.Cdi;
+import org.apache.openwebbeans.junit5.bean.MyService;
+import org.junit.jupiter.api.Test;
+
+@Cdi(classes = MyService.class)
+class ParameterResolutionTest
+{
+    @Test
```

Comment:
is it possible to add another bean/service and write a junit extension injecting it and a test method having a mixed cdi and not cdi injection for types being all cdi beans?

Reply:
That is possible if the other parameter resolver would know which parameter to resolve and which other one not, just like cdi extension. Is it really important and relevant to demonstrate in another test case?

Reply:
Not sure when you wrote the comment compared to when i looked but current test version is ok for me and tests what I wanted to harness, :+1: .

Reply:
Glad that we agree on that. As the test looks to me now I would prefer to remove `testMixedCdiAndOtherParameterResolver` again because in my opinion it has nothing to do with cdi in addition to the other tests but only asserts what we always knew about how JUnit handles parameter resolvers. But I can also live with it.

Reply:
Hmm, yes and no. It enables users to handle themselves the exclusions of parameters without @DontInject which is good when the extension resolves the parameter in CDI context but with additional rules (like adding an implicit qualifier or having a type mapping).
Since we have a DoneInject test, it is ok to drop but I would keep it as a doc example, can simplify things in some rare cases for "application junit extensions" - extension you write in final apps, not as a reusable extension/lib.

Reply:
Ok. Convinced as an example to document. When worth to document I guess we also can keep in the tests. The `OnlyFirstParameterResolver` might for that purpose be a little too minimalistic resolving only parameters that meet `parameterContext.getIndex() == 0` and then to the value null.
Is it correct that the documentation is behind that subversion link you posted somewhere and hence not in the git repository? If so, can we delay changes to the docs until after this here is merged?